### PR TITLE
Move from Travis to github actions

### PR DESCRIPTION
**Pull request by rejao**
Moving from Travis to github actions 

Tested in my repo - latest run here: https://github.com/rejao/flowgger/actions/runs/2663508256

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          default: true
+
+      - name: cargo fmt -- --check
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
+
+  test:
+    name: Test flowwger
```
Comment (kurojishi) [inline .github/workflows/ci.yml]: typo

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    runs-on: ubuntu-latest
+    needs: [style]
+    strategy:
+      matrix:
+        target: [
+          x86_64-unknown-linux-gnu,
+          x86_64-unknown-linux-musl,
+          i686-unknown-linux-gnu,
+          aarch64-unknown-linux-gnu,
```
Comment (kurojishi) [inline .github/workflows/ci.yml:43]: we want this arch to run for the musl toolchain as well if possible

**Review by kurojishi — CHANGES_REQUESTED** [review]
couple of minor comments

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vche) [reply .github/workflows/ci.yml:43]: +1

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rejao) [reply .github/workflows/ci.yml]: fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (rejao) [reply .github/workflows/ci.yml:43]: Done

**Review by kurojishi — APPROVED** [review]
ship it!

**Review by vche — APPROVED** [review]

**Pull request merged**